### DISCUSSION TITLE
V5 Docs typo

### DISCUSSION THREAD

**Discussion by jramiresbrito** [twbs/bootstrap#31267] [General]
Hey guys, I'm sorry if this isn't the correct channel to report this. *Please feel free to point where it should be.*

I want to report a litle typo that I found in v5 here:

[v5 docs - installing via yarn](https://v5.getbootstrap.com/docs/5.0/getting-started/download/#yarn)

it says:

```yarn add bootstrap```

But it isntall the v4.5.0

to properly install v5 please run:

```yarn add bootstrap@next```

*PS: There's also a problem with the ruby gem (it doesn't exists in rubygems yet)*

**Comment by mdo** [comment]
Issues are for bugs, Discussions are used for conversation :).
